src/ServiceControl.Monitoring.AcceptanceTests/ServiceControl.Monitoring.AcceptanceTests.csproj

```diff
@@ -42,6 +42,9 @@
       <HintPath>..\packages\NServiceBus.6.2.0\lib\net452\NServiceBus.Core.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="NServiceBus.Metrics, Version=1.0.0.0, Culture=neutral, PublicKeyToken=9fc386479f8a226c, processorArchitecture=MSIL">
```

Comment:
In [when_querying_timing_data](https://github.com/Particular/ServiceControl.Monitoring/pull/21/files#diff-46cce0a428a906b09e88829defd86bcfR53) to setup metric reporting in the monitored endpoint.